internal/fs/read_cache_test.go

```diff
@@ -731,8 +732,13 @@ type FileCacheWithDefaultCacheLocation struct {
 	fsTest
 }
 
+type FileCacheWithUserDefinedTempAsCacheLocation struct {
+	fsTest
+}
+
 func init() {
 	RegisterTestSuite(&FileCacheWithDefaultCacheLocation{})
+	RegisterTestSuite(&FileCacheWithUserDefinedTempAsCacheLocation{})
```

Comment:
@sethiay let me know if this needs to be registered in different init.. if its okay, should I add all register statements into one init method (better readable if they are all in one place)?

Reply:
Yes, adding at one place is better.

Reply:
done

Reply:
Thank you, nit: can we move init at the top - after constants definition and before first struct ?

Reply:
done